casper/src/main/scala/coop/rchain/casper/merging/DeployChainIndex.scala

```diff
@@ -71,8 +71,8 @@ object DeployChainIndex {
   }
 
   def deployChainCost(r: DeployChainIndex) = r.deploysWithCost.map(_.cost).sum
-  def isDependency(a: DeployChainIndex, dependencyFor: DeployChainIndex) =
-    EventLogMergingLogic.depends(dependencyFor.eventLogIndex, a.eventLogIndex)
+  def depends(a: DeployChainIndex, b: DeployChainIndex) =
+    EventLogMergingLogic.depends(a.eventLogIndex, b.eventLogIndex)
```

Comment:
Previously I messed up with this because I was confused when writing test. I had reference map with reverse direction (dependencies as keys), and deriving `depends` function. And there is no need to do the same here.
https://github.com/rchain/rchain/blob/dc9f4ddb99cbbfb8e70887d7cc43e5fd3b248394/sdk/src/test/scala/coop/rchain/sdk/merging/DagMergingLogicSpec.scala#L39-L45